cmd/snap/cmd_prepare_image.go

```diff
@@ -26,12 +26,13 @@ import (
 
 	"github.com/snapcore/snapd/i18n"
 	"github.com/snapcore/snapd/image"
+	"github.com/snapcore/snapd/osutil"
 )
 
 type cmdPrepareImage struct {
 	Positional struct {
-		ModelAssertionFn string `positional-arg-name:"model-assertion" description:"the model assertion name"`
-		Rootdir          string `long:"root-dir" description:"the output directory" `
+		ModelAssertion string `positional-arg-name:"model-assertion" description:"the model assertion name"`
```

Comment:
The documentation of this command would benefit from some love. The model assertion "name" isn't quite right whether this is a filename or a "reference" (which by itself seems bad).